first-party/jib-extension-3147-common/build.gradle

```diff
@@ -0,0 +1,8 @@
+repositories {
+  // because gradle plugin dependencies are pulling from jcenter
+  jcenter()
+}
+
+dependencies {
+  compileOnly "com.google.code.findbugs:jsr305:3.0.2"

```

Comment:
We shouldn't have a common project. The Maven and Gradle will be published as separate JARs, so unless we publish the common project as another separate JAR, having a common package will result in [split packages](https://www.logicbig.com/tutorials/core-java-tutorial/modules/split-packages.html).

Reply:
That is, we should duplicate the code in Maven and Gradle, and that's fine.